### ISSUE TITLE
[Bug]: Dialog - Using `aria-describedby="undefined"` triggers warning

### ISSUE THREAD

**Issue by Ericlm** [radix-vue/radix-vue#1262]
### Environment

Using Stackblitz for reproduction, after trying on macOS with Firefox.

### Link to minimal reproduction

https://stackblitz.com/edit/5nwtfn?file=src%2FApp.vue

### Steps to reproduce

1. Open the Dialog demo code
2. Removing the `<DialogDescription>` component in the template
3. Warning in the console, which is expected
4. Setting `aria-describedby="undefined"` in the `<DialogContent>`
5. Warning stills there

### Describe the bug

I would like to be able to build a Dialog without any `DialogDescription` inside (visible or not). The doc says that by putting `aria-describedby="undefined"` in DialogContent it should be ok. (https://www.radix-vue.com/components/dialog.html#description)

### Expected behavior

According to the docs, I was expecting a warning when removing it. But I expected the warning to go with the `aria-describedby="undefined"` on the `DialogContent`.

**Comment by Abdallah01win** [comment]
@Ericlm Was this issue solved for you after updating to V1.9.6, please? Because I have updated the package and I get the following warning: 
![warn](https://github.com/user-attachments/assets/8783c3c0-7383-4e12-99af-901c2a86a897)
And here's my code:
![Uploading code.png…]()

**Comment by Ericlm** [comment]
From the warning, I read that this is the DialogPortal that have "aria-describedby" attribute.

Everything is working well with the aria-describedby on the dialog content only 🙂

**Comment by Abdallah01win** [comment]
The base "DialogContent" component you get from 'Shad-cn/vue" uses the "DialogPortal" in its template. When passing "aria-describedby" to the "DialogContent" component it's also passed to the "DialogPortal" for some reason. I'm unsure if it's an issue or was intended as such.
